src/App.jsx

```diff
@@ -14,70 +15,90 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const goodsObject = goods
+  .map((good, index) => ({ name: good, id: index }));
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
```

Comment:
Usually when naming function which updates state, use the same name and `set` on the beginning, in your example it will be `[selectedGood, setSelectedGood]`, camelCase of course.